staff/dtiburcio/JS-Exercises/JavascriptExercises.js

```diff
@@ -0,0 +1,177 @@
+*/
+
+function translate(stringtoTranslate){
+    var stringTranslate = stringtoTranslate.split(' ');
+    var words = stringTranslate.length - 1;
+
+    for (var i = 0; i <= words; i++){
+        switch (stringTranslate[i]) {
+            case 'merry': 
+                stringTranslate[i] = 'god'; 
+                break;
+            case 'christmas':
+                stringTranslate[i] = 'jul';
+                break;
+            case 'and':
+                stringTranslate[i] = 'och';
+                break;
+            case 'happy':
+                stringTranslate[i] = 'gott';
+                break;
+            case 'new':
+                stringTranslate[i] = 'nytt';
+                break;
+            case 'year':
+                stringTranslate[i] = 'år';
+                break;
+            default:
+                break;
+          }
+    }
+    var result = stringTranslate.join();
+    var result2 = result.replace(/,/g,' ');
+    return result2;
+}
+
+/*
+findLongestWord()
+Write a function findLongestWord() that takes an array of words and returns the length of the longest one.
+*/
+
+function findLongestWord(stringToEvaluate){
+    var stringEvaluate = stringToEvaluate.split(' ');
+    var words = stringEvaluate.length - 1;
+    var result = '';
+    var max = 0;
+
+    for (var i = 0; i <= words; i++){
+        temp = stringEvaluate[i].length;
+        
+        if (temp > max){
+            result = stringEvaluate[i];
+            max = temp;
+        }
+    }
+    return max;
+}
+
+/*
+filterLongWords()
+Write a function filterLongWords() that takes an array of words and an integer i and returns the array of words that are longer than i.
+*/
+
+function filterLongWords(stringOfWords, mininum){
+    var listOfWords = stringOfWords.split(' ');
+    var words = listOfWords.length - 1;
+    var min = mininum;
+    console.log(min);
+    var result = [];
+
+    for (var i = 0; i <= words; i++){
+        if (listOfWords[i].length > min){
+            result.push(listOfWords[i]);
+        }
+    }
+    return result;
+}
+
+/*
+charFreq()
+Write a function charFreq() that takes a string and builds a frequency listing of the characters contained in it. Represent the frequency listing as a Javascript object. Try it with something like charFreq("abbabcbdbabdbdbabababcbcbab").
+*/
+
+function charFreq(stringEvaluate){
+    var stringLetters = stringEvaluate;
+    var letters = stringLetters.length - 1;
+
+    var freq = new Object();
+            freq.letraA = 0;
+            freq.letraB = 0;
+            freq.letraC = 0;   
+
+    for (var i = 0; i <= letters; i++){
+        if (stringLetters[i] == 'a'){
+            freq.letraA += 1;
+        }
+        if (stringLetters[i] == 'b'){
+            freq.letraB += 1;
+        }
+        if (stringLetters[i] == 'b'){
+            freq.letraC += 1;
+        }
+    }
+    return freq;
+}
```

Comment:
Will it work w/ `charFreq("fdsg gsfgsfg fdgdfgdfsgsfg")` or  `charFreq("sdv svpoierhwfds")`?  It should

Reply:
not functional... :(

Reply:
Solucionated!